src/pages/Channels.tsx

```diff
@@ -7,13 +7,27 @@ import CreateChannel from '../features/Channels/CreateChannel';
 import { selectUser } from '../features/User/userSlice';
 import { useAppSelector } from '../lib/store';
 
+const MAX_CHANNELS = 3; // Maximum channels per user
+
 const Channels = () => {
   const [openForm, setOpenForm] = useState(false);
   const user = useAppSelector(selectUser);
   const channels = useAppSelector((state) =>
     getCreatedChannels(state, user?.id ?? ""),
   );
 
+  const channelCount = channels.length;
+
+  const canCreateChannel = channelCount < MAX_CHANNELS;
+
+  const handleCreateChannelClick = () => {
+    if (canCreateChannel) {
+      setOpenForm(true);
+    } else {
+      alert("You've reached the maximum limit of channels (3).");
```

Comment:
Could you change this `alert` to `toast` for consistency and better UX?

Here's a sample use case from another component.

https://github.com/IndieCoderMM/convoz-chat/blob/c225c0b73e2d68e96607b6fae65cef13fdc7868a/src/features/Channels/ChannelCard.tsx#L34

Reply:
Yes I will do it in the next update